estate/models/estate_property.py

```diff
@@ -0,0 +1,107 @@
+from odoo import fields, models, api
+from odoo.exceptions import UserError , ValidationError
+
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    _description = "Test Property"
+    _order = "id desc"
+
+    name = fields.Char(required=True,string="Title")
+    salesperson_id = fields.Many2one('res.users',string="Salesman",default=lambda self: self.env.user)
+    buyer = fields.Many2one('res.partner',string="Buyer",readonly=True)
+    description = fields.Text()
+    postcode = fields.Char()
+    date_availability = fields.Date(copy=False)
+    expected_price = fields.Float(required=True)
+    selling_price = fields.Float(copy=False)
+    bedrooms = fields.Integer(default=2)
+    living_area = fields.Integer()
+    facades = fields.Integer()
+    active = fields.Boolean(default=True)
+    garage = fields.Boolean()
+    garden = fields.Boolean(default=True)
+    garden_area = fields.Integer()
+    garden_orientation = fields.Selection(
+        string='Garden Orientation',
+        selection=[
+           ('north', 'North'),
+           ('south', 'South'),
+           ('west', 'West'),
+           ('east', 'East')
+        ]
+    )
+    state = fields.Selection(
+        string='State',
+        required=True,
+        store=True,
+        copy=False,
+        default='new',
+        selection=[
+            ('new','New'),
+            ('offer_received','Offer Received'),
+            ('offer_accepted','Offer Accepted'),
+            ('sold','Sold'),
+            ('cancelled','Cancelled')
+        ]
+    ) 
+    property_type_id = fields.Many2one('estate.property.type',string="Property Type")
+    tag_ids = fields.Many2many("estate.property.tag",string="Property Tag")
```

Comment:
```suggestion
    tag_ids = fields.Many2many("estate.property.tag", string="Property Tag")
```
always leave one space after `,` and what will be the name of table for this many2many field? and if I want to change it how can I do it?

Reply:
the name of the table for this many2many field is estate_property_estate_property_tag_rel .  and this way we can change 
 property_tags = fields.Many2many(
        "estate.property.tag", 
        string="Property Tag", 
        relation="property_tag_rel",                    
        column1="estate_property_id",               
        column2="estate_property_tag_id"  
    )